dev/VSIX/Extension/Cpp/Common/VSPackage.resx

```diff
@@ -151,58 +151,58 @@
     <value>[Experimental] Resources File (.resw)</value>
   </data>
   <data name="1015" xml:space="preserve">
-    <value>A file for storing string and conditional resources for apps based on the Windows UI Library (WinUI 3).</value>
+    <value>A file for storing string and conditional resources (WinUI 3).</value>
   </data>
   <data name="1017" xml:space="preserve">
-    <value>Custom Control (WinUI 3)</value>
+    <value>Templated Control</value>
   </data>
   <data name="1018" xml:space="preserve">
-    <value>[Experimental] Custom Control (WinUI 3)</value>
+    <value>[Experimental] Templated Control</value>
   </data>
   <data name="1019" xml:space="preserve">
-    <value>A custom control for apps based on the Windows UI Library (WinUI 3).</value>
+    <value>A blank custom control with the appropriate default styling (WinUI 3).</value>
   </data>
   <data name="1021" xml:space="preserve">
-    <value>User Control (WinUI 3)</value>
+    <value>User Control</value>
   </data>
   <data name="1022" xml:space="preserve">
-    <value>[Experimental] User Control (WinUI 3)</value>
+    <value>[Experimental] User Control</value>
   </data>
   <data name="1023" xml:space="preserve">
-    <value>A user control for apps based on the Windows UI Library (WinUI 3).</value>
+    <value>A blank user control (WinUI 3).</value>
   </data>
   <data name="1025" xml:space="preserve">
-    <value>Blank App, Packaged with Windows Application Packaging Project (WinUI 3 in Desktop)</value>
+    <value>WinUI Blank App (Packaged with Windows Application Packaging Project)</value>
```

Comment:
I like the preceding strings that just have WinUI in parentheses
```suggestion
    <value>Blank App (WinUI, Packaged with Windows Application Packaging Project)</value>
```

Reply:
Why not just WinUI App?

Reply:
I'm fine with either WinUI Foo ... and Foo ... (WinUI) - just would like consistency